src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Designers/Input/AbstractProjectCommand.cs

```diff
@@ -52,8 +52,9 @@ public Task<bool> TryHandleCommandAsync(IImmutableSet<IProjectTree> nodes, long
         private static long[] GetCommandIds(AbstractProjectCommand command)
         {
             ProjectCommandAttribute attribute = (ProjectCommandAttribute)Attribute.GetCustomAttribute(command.GetType(), typeof(ProjectCommandAttribute));
-            if (attribute == null)
-                return Array.Empty<long>();
+
+            // All ProjectCommand's should be marked with [ProjectCommandAttribute]
+            Assumes.NotNull(attribute);
```

Comment:
Roslyn initiates a non-fatal Watson that doesn't crash the process but sends the dumps.